scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN apt-get -y update && \
 		software-properties-common \
 		sudo \
 		texinfo \
+		tree \
 		udev \
```

Comment:
will you update the .github/workflows/installer.yml  to remove the HACK in a subsequent patch?

Reply:
`.github/workflows/shallow_checkpatch.yml`, yes. Note this Dockerfile does not take effect automatically, someone has to rebuild and publish the image.